app/marketing/management/commands/sync_mail.py

```diff
@@ -91,17 +95,18 @@ def pull_to_db():
 
     print("- tdi")
     from tdi.models import WhitepaperAccess, WhitepaperAccessRequest
-    for wa in WhitepaperAccess.objects.all():
-        process_email(wa.email, 'whitepaperaccess')
+    for email in WhitepaperAccess.objects.filter(modified_on__gt=before_timestamp).values_list('email', flat=True):
+        process_email(email, 'whitepaperaccess')
 
-    for wa in WhitepaperAccessRequest.objects.all():
-        process_email(wa.email, 'whitepaperaccessrequest')
+    for email in WhitepaperAccessRequest.objects.filter(modified_on__gt=before_timestamp).values_list('email', flat=True):
+        process_email(email, 'whitepaperaccessrequest')
 
     print('/pull_to_db')
 
 def sync_mailchimp_list(eses, list_id):
     print("- {} emails".format(eses.count()))
-    for es in eses:
+    client = MailChimp(mc_api=settings.MAILCHIMP_API_KEY, mc_user=settings.MAILCHIMP_USER)
```

Comment:
@danlipert this just seemed like a bug, `client` was undefiend in this method before.